Swift/Tests/MessageEndpointTest.swift

```diff
@@ -77,7 +77,7 @@ class MessageEndpointTest: CBLTestCase, MCNearbyServiceBrowserDelegate,
 MCNearbyServiceAdvertiserDelegate, MCSessionDelegate, MessageEndpointDelegate,
 MultipeerConnectionDelegate {
     
-    var otherDB: Database!
+    var oDB: Database!
```

Comment:
name? since everywhere we named it otherDB, do you have any reason why named it oDB?

Reply:
It's for a convenience so that it is not optional (otherDB is optional in the base case).

`oDB = otherDB!`